src/data/components.json

```diff
@@ -917,7 +943,7 @@
       "description": "Content group — banner is designed as a subtle callout, intended for tangential information or announcements that are supplementary to the page's main focus.",
       "storybook": {
         "webcomponents": "http://ibmdotcom-web-components.mybluemix.net/?path=/story/components-content-group-banner--default",
-        "react": "http://ibmdotcom-react.mybluemix.net/?path=/story/components-content-group-banner--default"
+        "react": " "
```

Comment:
@shixiedesign Removing the space will automatically insert `Coming soon`. Currently with the space, an empty link will be inserted instead.

![image](https://user-images.githubusercontent.com/909118/118330158-5d394480-b4d5-11eb-8fe1-d5773494d40d.png)


```suggestion
        "react": ""
```